versions.json

```diff
@@ -0,0 +1,119 @@
+{
```

Comment:
Not a bad idea, I guess we can update the current scripts to utilize this going forward.